pkgs/development/python-modules/reproject/default.nix

```diff
@@ -12,39 +11,55 @@
 , pytest-astropy
 , setuptools-scm
 , cython
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "reproject";
-  version = "0.7.1";
+  version = "0.8";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1jsc3ad518vyys5987fr1achq8qvnz8rm80zp5an9qxlwr4zmh4m";
+    sha256 = "sha256-Z54sY3R6GViTvMLHrJclrAZ1dH4/9bzIrgqDd9nFbJY=";
   };
 
-  patches = [ (fetchpatch {
-      # Can be removed in next release after 0.7.1
-      # See https://github.com/astropy/reproject/issues/246
-      url = "https://github.com/astropy/reproject/pull/243.patch";
-      sha256 = "0dq3ii39hsrks0b9v306dlqf07dx0hqad8rwajmzw6rfda9m3c2a";
-    })
+  nativeBuildInputs = [
+    astropy-extension-helpers
+    astropy-helpers
+    cython
+    setuptools-scm
   ];
 
-  propagatedBuildInputs = [ numpy astropy astropy-healpix astropy-helpers scipy ];
-  nativeBuildInputs = [ astropy-helpers cython astropy-extension-helpers setuptools-scm ];
-  checkInputs = [ pytest pytest-astropy ];
+  propagatedBuildInputs = [
+    astropy
+    astropy-healpix
+    astropy-helpers
+    numpy
+    scipy
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-astropy
+  ];
 
   # Tests must be run in the build directory
   checkPhase = ''
     cd build/lib*
     pytest
   '';
```

Comment:
Please switch to pytestCheckHook and do the cd in preCheck.

Reply:
Using `pytestCheckHook` now.